Assets/BossRoom/Scripts/Client/Game/Action/AnimationOnlyActionFX.cs

```diff
@@ -10,7 +10,7 @@ public class AnimationOnlyActionFX : ActionFX
 
         public override bool Start()
         {
-            if( !Anticipated )
+            if (!Anticipated)
```

Comment:
mmm I would have hoped that this hole set of "Animation only" classes would disappear with networkAnimator :(